src/api/people.ts

```diff
@@ -0,0 +1,26 @@
+import { Person } from '../types/Person';
+import { PersonFromServer } from '../types/PersonFromServer';
+
+const API_URL
+  = 'https://mate-academy.github.io/react_people-table/api/people.json';
```

Comment:
USe interpolation here
```suggestion
const API_URL = `https://mate-academy.github.io/react_people-table/api/people.json`;
```